src/main/java/seedu/address/logic/commands/people/PeoplePaidCommand.java

```diff
@@ -0,0 +1,134 @@
+package seedu.address.logic.commands.people;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliPrefix.PEOPLE_COMMAND_TYPE;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.tag.Tag;
+import seedu.address.model.transaction.Amount;
+import seedu.address.model.transaction.Debt;
+import seedu.address.model.transaction.Loan;
+import seedu.address.model.transaction.TransactionList;
+
+/**
+ * Records that the person, who owe the user money has paid the user back.
+ */
+public class PeoplePaidCommand extends Command {
+
+    public static final String COMMAND_WORD = "paid";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Records that the person has paid you back.\n"
+            + "Parameters: <person's index> (must be a positive integer) "
+            + "[<loans's index> (must be a positive integer)]\n"
+            + "Example: " + PEOPLE_COMMAND_TYPE + " "
+            + COMMAND_WORD + " 4 1";
+
+    public static final String MESSAGE_PAID_SUCCESS = "Removed loan to %1$s by %2$s. %1$s now owes you %3$s.";
+
+    private final Index targetPersonIndex;
+    private Index targetLoanIndex;
+
+    public PeoplePaidCommand(Index targetIndex) {
```

Comment:
Actually dont even need this constructor. then line 42 can be final. 
This can be handled in PeoplePaidCommandParser. 

public PeopleReturnedCommand parse(String args) throws ParseException {
        requireNonNull(args);
        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_TRANSACTION_INDEX);

        Index personIndex;
        Index debtIndex;

        try {
            personIndex = ParserUtil.parseIndex(argMultimap.getPreamble());
        } catch (ParseException pe) {
            throw new ParseException(
                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, PeopleOweCommand.MESSAGE_USAGE), pe);
        }

        if (argMultimap.getValue(PREFIX_TRANSACTION_INDEX).isPresent()) {
            debtIndex = ParserUtil.parseIndex(argMultimap.getValue(PREFIX_TRANSACTION_INDEX).get());
        } else {
            debtIndex = null;
        }

        return new PeopleReturnedCommand(personIndex, debtIndex);

    }

Reply:
we have to add $/ in front of our loan index? what is the PREFIX_TRANSACTION_INDEX for?